src/Sulu/Bundle/MediaBundle/Resources/js/components/ImageFocusPoint/FocusPoint.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import {Icon} from 'sulu-admin-bundle/components';
+import type {Point} from './types';
+import focusPointStyles from './focusPoint.scss';
+
+const ARROW_UP_ICON = 'arrow-up';
+
+type Props = {
+    size: number,
```

Comment:
Do we need different sizes?

Reply:
No but you can't set that from outside anyway. I did it this way because I don't want define `FOCUS_POINT_MATRIX_SIZE` multiple times.